### PR TITLE
Elaborated on filter() and callback parameters

### PR DESCRIPTION
**Pull request by MatForsberg**
Updated description and added callback parameters

### CODE REVIEW

File: js-Array-prototype-filter.md [hunk 1]
```diff
@@ -6,17 +6,25 @@ The `filter()` method creates a new array with all elements that pass the test i
...
+| Required | arr       | Array being filtered                              |
+| Required | callback  | Function with 3 arguments (element, index, array) |
+| Optional | thisArg   | Value of *this* while executing callback          |
 
-**Return:** A new array that contains all the values for which the callback function returns true. If the callback function returns false for all elements of array1, the length of the new array is 0.
+**Return:** A new array that contains all the values for which the callback function returns true. If the callback function returns false for all elements of arr, the length of the new array is 0.
 
 [MDN link](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter) | [MSDN link](https://msdn.microsoft.com/library/ff679973%28v=vs.94%29.aspx?f=255&MSPPError=-2147217396)
+
+## callback Parameters
```
Comment (SaintPeter) [inline js-Array-prototype-filter.md]: Capital C on callback?

**Comment by SaintPeter** [comment]
Looks good, just needs a minor fix.

**Comment by Rafase282** [comment]
@MatForsberg Looks good, squash and it will be merged.

**Comment by MatForsberg** [comment]
I got it squashed. sorry for the confusion

**Pull request merged**